jerry-core/ecma/builtin-objects/ecma-builtin-promise.c

```diff
@@ -610,7 +609,7 @@ ecma_builtin_promise_race_or_all (ecma_value_t this_arg, /**< 'this' argument */
 
   if (ECMA_IS_VALUE_ERROR (ret))
   {
-    ret = JERRY_CONTEXT (error_value);
+    ret = jcontext_retrieve_exception ();
```

Comment:
Why jcontext? Do we have any functions in jcontext? What about `ecma_take_exception` ?

Reply:
> Why jcontext?

The global error is part of the Jerry context therefore IMO the `jcontext` prefix is more suitable than the `ecma`.

> Do we have any functions in jcontext?
Currently we do not have any functions there, but I think that is not a blocking issue.

Reply:
`jcontext_take_exception` ? Its an ownership taking.

Reply:
Sure! I can deal with this renaming.

Reply:
`jcontext_catch_exception` could be a good idea.

Reply:
I'd prefer the `jcontext_take_exception`.